package.json

```diff
@@ -37,9 +40,8 @@
     "vite": "^4.3.9"
   },
   "lint-staged": {
-    "*.{js,css,ts,tsx,jsx}": [
-      "prettier --write",
+    "*.{js,ts,tsx,jsx}": [
       "eslint --fix"
     ]
   }
-}
+}
```

Comment:
missing empty end line, I think it should be configured in your IDE :)

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file